### ISSUE TITLE
Dependencies on Linux Mint 17.3 (and presumably Ubuntu 14.04)

### ISSUE THREAD

**Issue by eliezerlp** [haakonnessjoen/MAC-Telnet#27]
Was getting:
```➜  haakonnessjoen-MAC-Telnet-18ed3bc ./autogen.sh 
aclocal: warning: couldn't open directory 'm4': No such file or directory
Can't exec "autopoint": No such file or directory at /usr/share/autoconf/Autom4te/FileUtils.pm line 345.
autoreconf: failed to run autopoint: No such file or directory
autoreconf: autopoint is needed because this package uses Gettext
```
Had to make a `m4` folder and install `autotools-dev` for it to succeed.

Also the instructions for installing on Linux from `README.markdown` seem to not work for me:
```➜  temp wget http://github.com/haakonnessjoen/MAC-Telnet/tarball/master
--2017-03-13 21:38:54--  http://github.com/haakonnessjoen/MAC-Telnet/tarball/master
Resolving github.com (github.com)... 192.30.253.112, 192.30.253.113
Connecting to github.com (github.com)|192.30.253.112|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://github.com/haakonnessjoen/MAC-Telnet/tarball/master [following]
--2017-03-13 21:38:54--  https://github.com/haakonnessjoen/MAC-Telnet/tarball/master
Connecting to github.com (github.com)|192.30.253.112|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://codeload.github.com/haakonnessjoen/MAC-Telnet/legacy.tar.gz/master [following]
--2017-03-13 21:38:54--  https://codeload.github.com/haakonnessjoen/MAC-Telnet/legacy.tar.gz/master
Resolving codeload.github.com (codeload.github.com)... 192.30.253.120, 192.30.253.121
Connecting to codeload.github.com (codeload.github.com)|192.30.253.120|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 66860 (65K) [application/x-gzip]
Saving to: ‘master’

100%[====================================================>] 66,860       405KB/s   in 0.2s   

2017-03-13 21:38:55 (405 KB/s) - ‘master’ saved [66860/66860]

➜  temp tar zxvf haakonness*.tar.gz
zsh: no matches found: haakonness*.tar.gz
➜  temp tar zxvf master           
haakonnessjoen-MAC-Telnet-18ed3bc/
[...]
haakonnessjoen-MAC-Telnet-18ed3bc/src/utlist.h
➜  temp```

**Comment by haakonnessjoen** [comment]
Hi,

Thanks for the feedback :)

Ubuntu and (therefore maybe also) mint already has mactelnet in the contrib package repository, so you could do "apt install mactelnet-client" or "apt install mactelnet-server" i think.

I will update the README with the autotools-dev requirement.

About the unpacking instructions, github must have changed their redirects so that the filename has changed. I will update that too :) Thanks!

**Comment by haakonnessjoen** [comment]
Fixed in ef7ad847
